prismicio/slice-machine
# Mm/video popover

## Context

User story:

Given I open Slice Machine for the first time with a Next project i see a hover card tooltip with a video in it

+ when I click on the video it plays… (not tracked)

+ when I click the button it closes the tooltip.

+ when I hover in and out it closes the tooltip.

+ when I click close it closes the tooltip

+ tracking ? Track only when the user clicks on the Tutorial links (exactly as it is today, no change)

To test: 

the above story

nuxt, nothing should have changed


[Ticket](https://linear.app/prismic/issue/DT-1479/aadev-i-have-styles-components-to-go-into-the-hover-card)

TODO: 
 + remove filler text


## Checklist before requesting a review
- [ ] I hereby declare my code ready for review.
- [ ] If it is a critical feature, I have added tests.
- [ ] The CI is successful.
- [ ] If there could backward compatibility issues, it has been discussed and planned.




## [OPT] Preview
